models/models.go

```diff
@@ -239,6 +239,7 @@ func NewTestEngine(x *xorm.Engine) (err error) {
 
 	x.SetMapper(core.GonicMapper{})
 	x.SetLogger(log.XORMLogger)
+	x.ShowSQL(true)
```

Comment:
What does this line do?

Reply:
For debug to show the SQL

Reply:
So should we remove it?

Reply:
This should probably only be set to true only when running Gitea in development mode